secator/cli.py

```
@@ -772,38 +772,52 @@ def workspace():
+			for child in sorted(reports_dir.iterdir()):
+				if child.is_dir():
+					workspaces_dict[child.name] = {'count': 0, 'path': str(child)}
+		json_reports = []
+		for root, _, files in os.walk(CONFIG.dirs.reports):
+			for file in files:
+				if file.endswith('report.json'):
+					path = Path(root) / file
+					json_reports.append(path)
+		json_reports = sorted(json_reports, key=lambda x: x.stat().st_mtime, reverse=False)
+		for path in json_reports:
+			ws, runner_type, number = str(path).split('/')[-4:-1]
```

Ruff (0.15.15)

[warning] Unpacked variable `runner_type` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)